lib/Console/Display/ReportListDisplay.php

```diff
@@ -18,8 +18,9 @@ public function render(OutputFormatterInterface $formatter, Status $status): str
             $statuses[] = sprintf(
                 $this->resolveFormat($index + 1 === count($status->reportStore()), $report),
                 sprintf(
-                    '[%3s] %s',
+                    '[%3s][%s] %s',
```

Comment:
Is this useful information? (in relation to the other information)

Reply:
I'm not sure. I found it quite helpful while testing.